stdlib/2.7/socket.pyi

```diff
@@ -278,7 +278,7 @@ class socket:
     proto = 0
 
     def __init__(self, family: int = ..., type: int = ...,
-                 proto: int = ..., fileno: int = ...) -> None: ...
+                 proto: int = ..., fileno: Optional[int] = ...) -> None: ...
```

Comment:
Hm... fileno actually doesn't exist at all in Python 2, so please just delete this arg! (In Python 3 it is as you wrote.)